spec/features/viewing_peep_spec.rb

```diff
@@ -0,0 +1,13 @@
+feature 'Viewing Peeps' do
+    scenario 'User can view Peeps' do
+        Peep.create("I am having a great day")
+        Peep.create("I wonder what the weather is like tomorrow")
+        Peep.create("OMG I forgot my keys!")
+
+        visit '/peep'
+       
+        expect(page).to have_content "I am having a great day"
+        expect(page).to have_content  "I wonder what the weather is like tomorrow"
+        expect(page).to have_content "OMG I forgot my keys!"
```

Comment:
Nice idea adding several peeps to double-check that the page is behaving as it should when multiple peeps are present